scripts/export-dos-outcomes.py

```diff
@@ -77,10 +94,13 @@ def get_outcomes_locations(content_manifest):
 
     pool = ThreadPool(3)
 
+    logger.info(f"Finding suppliers for Digital Outcomes on {FRAMEWORK_SLUG}")
     suppliers = find_all_outcomes(client, map_impl=pool.imap)
 
+    logger.info(f"Building CSV for {len(suppliers)} Digital Outcomes suppliers")
     write_csv_with_make_row(
         suppliers,
         make_row(capabilities, locations),
-        "output/{}-outcomes.csv".format(FRAMEWORK_SLUG)
+        os.path.join(OUTPUT_DIR, "digital-outcomes-suppliers.csv".format(FRAMEWORK_SLUG)),
```

Comment:
There isn't a site for the format method to insert the framework slug in this string?

Reply:
Good spot. I thought I'd removed all those but that got left behind 😸